rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -271,15 +276,15 @@ def test_calculate_spend_cost_basis_sell_more_than_bought_after_year(accountant)
     asset = A_BTC
     cost_basis = accountant.pots[0].cost_basis
     asset_events = cost_basis.get_events(asset)
-    asset_events.acquisitions.append(
+    asset_events.acquisitions_manager.add_acquisition(
         AssetAcquisitionEvent(
             amount=FVal(1),
```

Comment:
Is a bit of nitpick but we have a constant for this

```suggestion
            amount=ONE,
```

Reply:
Okay